cmd/analyseError_test.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/damianoneill/nc-hammer/result"
+	"github.com/stretchr/testify/assert"
+)
+
+var myCmd = analyseErrorCmd
+
+func Test_Args(t *testing.T) {
+	t.Run("len(arg) != 1", func(t *testing.T) {
```

Comment:
the description for the test - len(arg) != 1, describes what you test but it doesnt convey what its means, it would be better to name the label 'test that analyse passes a directory to the command'